### DISCUSSION TITLE
Is there any possibility of Scala.js support?

### DISCUSSION THREAD

**Discussion by SethTisue** [lampepfl/gears#69] [General]
question says it all

**Comment by bishabosha** [comment]
Whoever answers there is also the same question on `/r/scala` : https://www.reddit.com/r/scala/comments/1cb88t8/gears_library_in_scalajs/

**Comment by armanbilge** [comment]
Maybe with JavaScript generators / `yield`.
- https://github.com/scala-js/scala-js/issues/5079

**Comment by SethTisue** [comment]
relevant: https://github.com/scala/scala3/issues/22474
